.ci/cico_build_prcheck_dep.sh

```diff
@@ -15,6 +15,13 @@ DOCKER_IMAGE="rh-che-automation-dep"
 DOCKER_IMAGE_URL="${REGISTRY}/openshiftio/${NAMESPACE}-${DOCKER_IMAGE}"
 TAG="latest"
 
+cat jenkins-env \
+    | grep -E "(ghprbSourceBranch|ghprbPullId|QUAY)=" \
```

Comment:
What do we need for the `ghprbSourceBranch` and `ghprbPullId`?

Reply:
they are not used, will remove.